metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -284,6 +284,7 @@ private Throwable handleEventException(String name,
             "Reverting to last committed offset {}.",
             this, exception.getClass().getSimpleName(), curClaimEpoch, deltaUs,
             lastCommittedOffset, exception);
+        raftClient.resign(curClaimEpoch);
```

Comment:
Do we also need to resign and renounce in line 272 of this file?

Reply:
@jsancio For 272 to execute, startProcessingTime must not be present. The only place I see an exception get thrown before startProcessingTime is defined is in the ControllerWriteEvent and it is a NotControllerException. This would mean that we would not have to resign/renounce since it is already not the controller.

Reply:
Forwarding write requests to the controller is a requirement for KRaft